tests/components/automation/test_state.py

```diff
@@ -130,25 +130,6 @@ def test_if_fires_on_attribute_change_with_to_filter(self):
         self.hass.block_till_done()
         self.assertEqual(1, len(self.calls))
 
-    def test_if_fires_on_entity_change_with_state_filter(self):
-        """Test for firing on entity change with state filter."""
-        assert setup_component(self.hass, automation.DOMAIN, {
-            automation.DOMAIN: {
-                'trigger': {
-                    'platform': 'state',
-                    'entity_id': 'test.entity',
-                    'state': 'world'
```

Comment:
That test already exists in `test_if_fires_on_entity_change_with_to_filter` or am I missing something?